src/lib/bottom-sheet/bottom-sheet-ref.ts

```diff
@@ -42,6 +45,7 @@ export class MatBottomSheetRef<T = any, R = any> {
     // @breaking-change 8.0.0 `_location` parameter to be removed.
     _location?: Location) {
     this.containerInstance = containerInstance;
+    this.disableClose = containerInstance.bottomSheetConfig.disableClose;
```

Comment:
Will this potentially change the behavior since you are setting the value in the class instead of reading it from the object at the time of the check?

Reply:
It won't change the behavior, because the old approach only read it once on init.

Reply:
Ahh, yes because it set up the subscription based on that. Makes sense.